lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
Ah, that's interesting. I don't think we've touched the definition of that method, but it's just returning the body of the result that `resin-request` returns, and that did have some big changes a little while back. None of the parts of the SDK that talk to the supervisor are tested, since they're tough to integration test, so it's hard to know when that broke.

I'll have a look into it now.